README.md

```diff
@@ -18,78 +66,47 @@ Use the hosted package:
 }
 ```
 
-Wrap your function handler
-```
-'use strict';
+## Step 2: Locate and set the ingest endpoint
 
-const signalFxLambda = require('signalfx-lambda');
+By default, this function wrapper will send data to the us0 realm. As a result, if you are not in us0 realm and you want to use the ingest endpoint directly, then you must explicitly set your realm. To set your realm, use a subdomain, such as ingest.us1.signalfx.com or ingest.eu0.signalfx.com.
 
-exports.handler = signalFxLambda.wrapper((event, context, callback) => {
-  ...
-});
-```
+To locate your realm:
 
-Using async/await,
-```
-'use strict';
+1. Open SignalFx and in the top, right corner, click your profile icon.
+2. Click **My Profile**.
+3. Next to **Organizations**, review the listed realm.
 
-const signalFxLambda = require('signalfx-lambda');
 
-exports.handler = signalFxLambda.asyncWrapper(async (event, context) => {
-  ...
-});
-```
-
-#### Configuring the ingest endpoint
-
-By default, this function wrapper will send to the `us0` realm. If you are
-not in this realm you will need to set the `SIGNALFX_INGEST_ENDPOINT` environment
-variable to the correct realm ingest endpoint (https://ingest.{REALM}.signalfx.com).
-To determine what realm you are in, check your profile page in the SignalFx
-web application (click the avatar in the upper right and click My Profile).
+## Step 3: Set environment variables
 
-
-### Environment Variables
+1. Set SIGNALFX_ACCESS_TOKEN with your correct access token. Review the following example. 
 
 ```
- SIGNALFX_AUTH_TOKEN=signalfx token
+SIGNALFX_ACCESS_TOKEN=access token
 ```
 
-Optional parameters available:
-```
-SIGNALFX_SEND_TIMEOUT=milliseconds for signalfx client timeout [1000]
+2. If you use POPS, Smart Gateway, or want to ingest directly from a realm other than us0, then you must set at least one endpoint variable. (For environment variables, SignalFx defaults to the us0 realm. As a result, if you are not in the us0 realm, you may need to set your environment variables.) You can update one of the variables below. Review the following examples.  
```

Comment:
As above, unfortunately each wrapper has a bit different configuration.
In case of nodejs the only variable available to set the ingest endpoint is SIGNALFX_INGEST_ENDPOINT. (which still needs to be updated for realms other than us0 and remarks about Smart Gateway and pop hold)

Reply:
Please review Step 3. I'm not very confident of how accurate it is. I was especially confused because in the master version, it says "SIGNALFX_INGEST_ENDPOINT=[https://pops.signalfx.com]." I would have assumed it would point to the gateway, not pops.